src/app/lecture/knowledge-node/notes/notes.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { Note } from './model/note.model';
+
+@Component({
+  selector: 'cc-notes',
+  templateUrl: './notes.component.html',
+  styleUrls: ['./notes.component.css']
+})
+export class NotesComponent implements OnInit {
+
+  text = '# Note';
```

Comment:
I don't think we need this initial text. Can we go with an empty string for this and the reset?